build-config/make/busybox.make

```diff
@@ -4,6 +4,7 @@
 #  Copyright (C) 2014 david_yang <david_yang@accton.com>
 #  Copyright (C) 2014 Mandeep Sandhu <mandeep.sandhu@cyaninc.com>
 #  Copyright (C) 2014 Nikolay Shopik <shopik@inblock.ru>
+#  Copyright (C) 2016 Pankaj Bansal <pankajbansal3073@gmail.com>
 #
```

Comment:
Moving the version of busybox forward is generally a good thing.

My concern here is that the new busybox version introduced here (1.24.2) does not include all the feature patches from the older version.  We had this same problem for u-boot, where folks wanted to use different versions of u-boot.

We have enhanced busybox with a number of feature patches.  These patches should also be applied to a newer version.  For example any ONIE compatible system will need all the "feature-onie-syseeprom" patches.

For u-boot we solved this by having a "common" patch directory that applied to all u-boot versions.  Otherwise we have to maintain patches for both busybox versions separately.  That is painful.